src/test/mir-opt/const-promotion-extern-static/rustc.BAR.PromoteTemps.diff

```diff
@@ -57,8 +57,12 @@
 -         return;                          // bb2[2]: scope 0 at $DIR/const-promotion-extern-static.rs:8:1: 8:53
 +         return;                          // bb2[0]: scope 0 at $DIR/const-promotion-extern-static.rs:8:1: 8:53
       }
-  }
+- }
 - 
-- alloc0: DefId(0:5 ~ const_promotion_extern_static[317d]::Y[0]): 4 bytes, alignment 4
-- 2a 00 00 00 
+- alloc0 {
+-     size: 4
+-     alignment: 4
+-     static const_promotion_extern_static[317d]::Y[0]
```

Comment:
It's unclear what this means. Also for symmetry it should likely be `static:`.

But I didn't mind the one-line presentation before, so what about
```
alloc0 (const_promotion_extern_static[317d]::Y[0]): size 4, alignment 4
{ 2a 00 00 00  }
```
```

Reply:
What I had in mind was more like:
```rust
alloc0 (const_promotion_extern_static::Y): size 4, align 4 {
    2a 00 00 00
}
```

Reply:
done